src/test/java/teammates/test/cases/pagedata/StudentHomePageDataTest.java

```diff
@@ -107,26 +110,31 @@ private void testOldCourseTable(CourseDetailsBundle oldCourse, CourseTable cours
 
         testFeedbackSession(index++, publishedRow, publishedSession,
                             Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_PENDING
-                                + Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_CLOSED
-                                + Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_PUBLISHED,
-                            "Published");
+                                + Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_CLOSED,
+                            Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_PUBLISHED,
+                            "Closed", "Yes");
         testFeedbackSession(index++, closedRow, closedSession,
                             Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_PENDING
                                 + Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_CLOSED,
-                            "Closed");
+                            Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_NOT_PUBLISHED,
+                            "Closed", "No");
         testFeedbackSession(index++, submittedClosedRow, submittedClosedSession,
                             Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_SUBMITTED
                                 + Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_CLOSED,
-                            "Closed");
+                            Const.Tooltips.STUDENT_FEEDBACK_SESSION_STATUS_NOT_PUBLISHED,
+                            "Closed", "No");
```

Comment:
@chowyb Ahh, I undertand your confusion! 😅 

If look closely, this is the `StudentHomePageDataTest`. As per @damithc's [suggestion](https://github.com/TEAMMATES/teammates/pull/7965#issuecomment-323556782), I had changed it to `published/not published` only for the Instructor Pages. Student Home Page still looks the same as the screenshot posted under the StudentHomePage heading [here](https://github.com/TEAMMATES/teammates/pull/7965#issuecomment-323538063). Hence, this test still works.